src/wp-admin/css/common.css

```diff
@@ -2433,6 +2433,12 @@ h1.nav-tab-wrapper, /* Back-compat for pre-4.4 */
 .nav-menus-php .metabox-holder h3 {
 	padding: 0;
 }
+
+.accordion-container h3.accordion-section-title {
+	padding: 0 !important;
```

Comment:
Curious why `!important` is needed here? Any chance someone is intentionally styling these titles in a way that would conflict with this specificity change?

Reply:
The previous styling was on the heading; the styling is now on the button inside the heading. A lot of previous styling was attached to headings. `!important` may not be the best option here, but I found there were so many different cases that needed to be overridden, it just became the easier path. If anybody wants to apply custom styles in this case, they *should* now be applying them to the contained button, so making this difficult to override makes sense.